### DISCUSSION TITLE
How to configure intellisense/plugins

### DISCUSSION THREAD

**Discussion by 999gs** [NvChad/NvChad#687] [General]
Hello, I have a question to load the python plugin, I added the plugin in **.config/nvim/lua/plugins/init.lua.**

![image](https://user-images.githubusercontent.com/96792586/147597638-c1cb8dab-7c1c-40db-9306-ef7089e99b40.png)

Then I installed and updated it

- PackerInstall
- PackerUpdate

Then I verified that the plugin is installed doing a **PackerStatus**

![image](https://user-images.githubusercontent.com/96792586/147597782-dc372a94-22b5-47ce-a7a8-d7ac64e7eb4f.png)

But this one is still not working for me

![Peek 2021-12-28 13-11](https://user-images.githubusercontent.com/96792586/147598080-e7ae23e2-6079-49a9-821c-cd93f64c310f.gif)

I don't know if I have to do another additional step or if I missed something.

Any help you can give me is appreciated.

Thanks!!! 👍️

**Comment by siduck** [comment]
why that plugin when you have treesitter? use :TSInstall python

**Reply by 999gs** [reply]
I used TSInstall and it still doesn't work for me.


- **TSInstallInfo**

 
![image](https://user-images.githubusercontent.com/96792586/147619037-a7b1a636-3440-496b-84f1-cc69715df7d6.png)

- **./plugin/packer_compiled.lua**

![image](https://user-images.githubusercontent.com/96792586/147619100-4bdd978a-ae68-4105-9fdf-6a94946509f7.png)

**Reply by siduck** [reply]
remove python syntax plugin first then check with treesitter

**Reply by 999gs** [reply]
Done, I removed it.

After I did that, I added it back again.

![TsInstallInfo](https://user-images.githubusercontent.com/96792586/147620979-d1efdee1-07ce-4754-bc09-149fe0e9c6df.png)
![modules](https://user-images.githubusercontent.com/96792586/147620976-5cafafd3-25e3-43d0-b90b-ad57e92d8489.png)


![image](https://user-images.githubusercontent.com/96792586/147621015-85ed108a-fa44-44ca-8a36-59a557fda436.png)

**Comment by 999gs** [comment]
It still does not work, what should I do in this case? @siduck 

Thanks!!!

**Reply by siduck** [reply]
> It still does not work, what should I do in this case? @siduck
> 
> Thanks!!!

Idk :/ , tressitter parsers work ootb. Lemme test this.

**Reply by 999gs** [reply]
![image](https://user-images.githubusercontent.com/96792586/147775394-66abb563-6bae-4045-8ce7-e319772d4173.png)

**Reply by 999gs** [reply]
I still haven't been able to figure out what's going on with intellisense python, but I've had fun reading and learning lua and making configuration changes. 👨‍💻🤣

![image](https://user-images.githubusercontent.com/96792586/147803882-1f239134-9d02-4a27-94f6-34b9d256e12d.png)

**Reply by 999gs** [reply]
Solved it!
